packages/fondue/src/components/RichTextEditor/Plugins/MentionPlugin/MentionCombobox/MentionCombobox.tsx

```diff
@@ -0,0 +1,38 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MentionComboboxItem } from '@components/RichTextEditor/Plugins/MentionPlugin/MentionCombobox/MentionComboboxItem';
+import { Combobox } from '@components/RichTextEditor/components/ComboBox/Combobox';
+import {
+    ComboboxProps,
+    ELEMENT_MENTION,
+    MentionPlugin,
+    getMentionOnSelectItem,
+    getPluginOptions,
+    useEditorRef,
+} from '@udecode/plate';
+
+export function MentionCombobox({
+    pluginKey = ELEMENT_MENTION,
+    id = pluginKey,
+    ...props
+}: Partial<ComboboxProps> & {
+    pluginKey?: string;
+}) {
+    const editor = useEditorRef();
+
```

Comment:
Mentions seem to be broken at the minute. I got this error when trying to select a person from the dropdown:
`
Error: Cannot find a descendant at path [3,1] in node: {"children":[{"type":"p","children":[{"text":"This text is bold.","textStyle":"p","bold":true}]},{"type":"p","children":[{"text":"This text is italic.","textStyle":"p","italic":true}]},{"type":"p","children":[{"text":"This text has an underline.","textStyle":"p","underline":true}]},{"type":"p","children":[{"text":"This text has a strikethrough.","textStyle":"p","strikethrough":true}]},{"type":"p","children":[{"text":"This text is a code line.","textStyle":"p","code":true}]},{"type":"p","children":[{"text":"An example of subscript is N"},{"text":"2","subscript":true}]},{"type":"p","children":[{"text":"An example of superscript is 2"},{"text":"5","superscript":true}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":""}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.","textStyle":"p"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number two.","textStyle":"custom1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number one.","textStyle":"custom2"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number three.","textStyle":"custom3"}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two, with more children."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number two."}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This comes first."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This comes second."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"And last but not least, this comes third."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 2"}]}]}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]}]},{"type":"p","children":[{"text":""},{"type":"a","children":[{"text":"This is a Link. Without any breaks before or after"}],"url":"https://frontify.com"},{"text":""}]},{"type":"p","children":[{"text":"Try mentioning characters, like "},{"type":"mention","category":"user","id":"aaaaaaaa20","children":[{"text":""}]},{"text":" or "},{"type":"mention","category":"group","id":"1111111111","children":[{"text":""}]},{"text":"."}]},{"type":"p","children":[{"text":""},{"type":"a","chosenLink":{"searchResult":{"link":"https://smartive.ch"},"openInNewTab":true},"children":[{"text":"This is also a Link (Legacy Format)."}]},{"text":""}]},{"type":"heading1","children":[{"text":"Heading 1","textStyle":"heading1"}]},{"type":"heading2","children":[{"text":"Heading 2","textStyle":"heading2"}]},{"type":"heading3","children":[{"text":"Heading 3","textStyle":"heading3"}]},{"type":"heading4","children":[{"text":"Heading 4","textStyle":"heading4"}]},{"type":"custom1","children":[{"text":"Custom 1","textStyle":"custom1"}]},{"type":"custom2","children":[{"text":"Custom 2","textStyle":"custom2"}]},{"type":"custom3","children":[{"text":"Custom 3","textStyle":"custom3"}]},{"type":"quote","children":[{"text":"Quote","textStyle":"quote"}]},{"type":"checkbox-item","checked":true,"indent":0,"children":[{"text":"This is a checked checklist item."}]},{"type":"checkbox-item","checked":false,"indent":0,"children":[{"text":"This is an unchecked checklist item."}]},{"type":"checkbox-item","checked":true,"indent":0,"children":[{"text":"This is checked again."}]},{"type":"checkbox-item","checked":true,"indent":0,"align":"right","children":[{"text":"This is aligned right and checked."}]}],"operations":[],"selection":{"anchor":{"path":[7,1,0,0],"offset":540},"focus":{"path":[7,0,0,0],"offset":0}},"marks":null,"id":"id-465466933","prevSelection":{"focus":{"path":[8,0],"offset":12}},"isFallback":false,"currentKeyboardEvent":null,"plate":{"set":{}},"key":0.7459577658487755,"plugins":[{"key":"react","type":"react","options":{},"inject":{},"editor":{}},{"key":"history","type":"history","options":{},"inject":{},"editor":{}},{"key":"nodeFactory","type":"nodeFactory","options":{},"inject":{},"editor":{}},{"key":"event-editor","handlers":{},"type":"event-editor","options":{},"inject":{},"editor":{}},{"key":"inline-void","type":"inline-void","options":{},"inject":{},"editor":{}},{"key":"insertData","type":"insertData","options":{},"inject":{},"editor":{}},{"key":"prevSelection","handlers":{},"type":"prevSelection","options":{},"inject":{},"editor":{}},{"key":"deserializeHtml","editor":{"insertData":{"format":"text/html"}},"_thenReplaced":1,"type":"deserializeHtml","options":{},"inject":{}},{"key":"deserializeAst","editor":{"insertData":{"format":"application/x-slate-fragment"}},"type":"deserializeAst","options":{},"inject":{}},{"key":"editorProtocol","type":"editorProtocol","options":{},"inject":{},"editor":{}},{"key":"softBreak","handlers":{},"options":{"rules":[{"hotkey":"shift+enter"}]},"type":"softBreak","inject":{},"editor":{}},{"key":"heading1","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h1","H1"]}]},"type":"heading1","options":{},"inject":{},"editor":{}},{"key":"heading2","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h2","H2"]}]},"type":"heading2","options":{},"inject":{},"editor":{}},{"key":"heading3","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h3","H3"]}]},"type":"heading3","options":{},"inject":{},"editor":{}},{"key":"heading4","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h4","H4"]}]},"type":"heading4","options":{},"inject":{},"editor":{}},{"key":"p","isElement":true,"handlers":{},"options":{"hotkey":["mod+opt+0","mod+shift+0"]},"deserializeHtml":{"rules":[{"validNodeName":"P"}]},"type":"p","inject":{},"editor":{}},{"key":"custom1","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom1"}]},"type":"custom1","options":{},"inject":{},"editor":{}},{"key":"custom2","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom2"}]},"type":"custom2","options":{},"inject":{},"editor":{}},{"key":"custom3","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom3"}]},"type":"custom3","options":{},"inject":{},"editor":{}},{"key":"imageTitle","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"imageTitle"}]},"type":"imageTitle","options":{},"inject":{},"editor":{}},{"key":"imageCaption","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"imageCaption"}]},"type":"imageCaption","options":{},"inject":{},"editor":{}},{"key":"quote","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["blockquote","BLOCKQUOTE"]}]},"type":"quote","options":{},"inject":{},"editor":{}},{"key":"combobox","handlers":{},"type":"combobox","options":{},"inject":{},"editor":{}},{"key":"mention","options":{"trigger":"@","insertSpaceAfterMention":true,"triggerPreviousCharPattern":{},"id":"mention"},"isElement":true,"isInline":true,"isVoid":true,"isMarkableVoid":true,"handlers":{},"plugins":[{"key":"mention_input","isElement":true,"isInline":true,"type":"mention_input","options":{},"inject":{},"editor":{}}],"_thenReplaced":2,"type":"mention","inject":{},"editor":{}},{"key":"mention_input","isElement":true,"isInline":true,"type":"mention_input","options":{},"inject":{},"editor":{}},{"deserializeHtml":{"rules":[{"validNodeName":["STRONG","B"]},{"validStyle":{"fontWeight":["600","700","bold"]}},{"validNodeName":["SPAN"],"validClassName":"tw-font-bold"}]},"options":{"hotkey":["mod+b","ctrl+b"]},"key":"bold","isLeaf":true,"handlers":{},"type":"bold","inject":{},"editor":{}},{"deserializeHtml":{"rules":[{"validNodeName":["EM","I"]},{"validStyle":{"fontStyle":"italic"}},{"validNodeName":["SPAN"],"validClassName":"tw-italic"}]},"options":{"hotkey":["mod+i","ctrl+i"]},"key":"italic","isLeaf":true,"handlers":{},"type":"italic","inject":{},"editor":{}},{"deserializeHtml":{"rules":[{"validNodeName":["U"]},{"validStyle":{"textDecoration":["underline"]}},{"validNodeName":["SPAN"],"validClassName":"tw-underline"}]},"options":{"hotkey":["mod+u","ctrl+u"]},"key":"underline","isLeaf":true,"handlers":{},"type":"underline","inject":{},"editor":{}},{"key":"strikethrough","isLeaf":true,"handlers":{},"options":{"hotkey":"mod+shift+x"},"deserializeHtml":{"rules":[{"validNodeName":["S","DEL","STRIKE"]},{"validStyle":{"textDecoration":"line-through"}}]},"type":"strikethrough","inject":{},"editor":{}},{"key":"a","deserializeHtml":{"rules":[{"validNodeName":"A"}]},"isElement":true,"isInline":true,"options":{"rangeBeforeOptions":{"matchString":" ","skipInvalid":true,"afterMatch":true},"triggerFloatingLinkHotkeys":"command+k, ctrl+k","keepSelectedTextOnPaste":true},"_thenReplaced":3,"type":"a","inject":{},"editor":{}},{"key":"code","isLeaf":true,"deserializeHtml":{"rules":[{"validNodeName":["CODE"]},{"validStyle":{"fontFamily":"Consolas"}}]},"handlers":{},"options":{"hotkey":"mod+e"},"type":"code","inject":{},"editor":{}},{"key":"breakAfterColumn","handlers":{},"options":{"columns":1,"gap":"normal","hotkey":["shift+ctrl+enter"]},"type":"breakAfterColumn","inject":{},"editor":{}},{"options":{"hotkey":["mod+.","ctrl+."],"clear":"superscript"},"key":"subscript","isLeaf":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":["SUB"]},{"validStyle":{"verticalAlign":"sub"}}]},"type":"subscript","inject":{},"editor":{}},{"options":{"hotkey":["mod+,","ctrl+,"],"clear":"subscript"},"key":"superscript","isLeaf":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":["SUP"]},{"validStyle":{"verticalAlign":"super"}}]},"type":"superscript","inject":{},"editor":{}},{"key":"align","inject":{"props":{"validTypes":["heading1","heading2","heading3","heading4","custom1","custom2","custom3","quote","imageCaption","imageTitle","p","p","lic","checkbox-item"],"nodeKey":"align","defaultNodeValue":"left","styleKey":"textAlign","validNodeValues":["left","center","right","justify"]},"pluginsByKey":{"heading1":{"deserializeHtml":{}},"heading2":{"deserializeHtml":{}},"heading3":{"deserializeHtml":{}},"heading4":{"deserializeHtml":{}},"p":{"deserializeHtml":{}},"custom1":{"deserializeHtml":{}},"custom2":{"deserializeHtml":{}},"custom3":{"deserializeHtml":{}},"imageTitle":{"deserializeHtml":{}},"imageCaption":{"deserializeHtml":{}},"quote":{"deserializeHtml":{}}}},"_thenReplaced":3,"type":"align","options":{},"editor":{}},{"options":{"isSoftBreak":false},"key":"list","plugins":[{"key":"ul","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"UL"}]},"type":"ul","options":{},"inject":{},"editor":{}},{"key":"ol","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"OL"}]},"type":"ol","options":{},"inject":{},"editor":{}},{"key":"li","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":"LI"}]},"_thenReplaced":3,"type":"li","options":{},"inject":{},"editor":{}},{"key":"lic","isElement":true,"type":"lic","options":{},"inject":{},"editor":{}}],"type":"list","inject":{},"editor":{}},{"key":"ul","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"UL"}]},"type":"ul","options":{},"inject":{},"editor":{}},{"key":"ol","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"OL"}]},"type":"ol","options":{},"inject":{},"editor":{}},{"key":"li","inject":{"pluginsByKey":{"deserializeHtml":{"editor":{"insertData":{}}}}},"isElement":true,"deserializeHtml":{"rules":[{"validNodeName":"LI"}]},"_thenReplaced":3,"type":"li","options":{},"editor":{}},{"key":"lic","isElement":true,"type":"lic","options":{},"inject":{},"editor":{}},{"key":"textStyle","isLeaf":true,"type":"textStyle","options":{},"inject":{},"editor":{}},{"key":"checkbox-item","isElement":true,"type":"checkbox-item","options":{},"inject":{},"editor":{}},{"key":"indent","inject":{"props":{"validTypes":["checkbox-item"],"nodeKey":"indent","styleKey":"marginLeft"}},"handlers":{},"options":{"offset":24,"unit":"px"},"_thenReplaced":3,"type":"indent","editor":{}},{"key":"emoji","options":{"id":"emoji","trigger":":","emojiTriggeringController":{"options":{"trigger":":","limitTriggeringChars":2},"_isTriggering":false,"_hasTriggeringMark":false,"text":""}},"_thenReplaced":3,"type":"emoji","inject":{},"editor":{}},{"key":"reset-formatting","isElement":false,"type":"reset-formatting","options":{},"inject":{},"editor":{}},{"options":{"rules":[{"mode":"text","match":"--","format":"—"},{"mode":"text","match":"...","format":"…"},{"mode":"text","match":">>","format":"»"},{"mode":"text","match":"<<","format":"«"},{"mode":"text","match":"->","format":"→"},{"mode":"text","match":"<-","format":"←"},{"mode":"text","match":"=>","format":"⇒"},{"mode":"text","match":["<=","≤="],"format":"⇐"},{"mode":"text","match":["(tm)","(TM)"],"format":"™"},{"mode":"text","match":["(r)","(R)"],"format":"®"},{"mode":"text","match":["(c)","(C)"],"format":"©"},{"mode":"text","match":"~0","format":"₀"},{"mode":"text","match":"~1","format":"₁"},{"mode":"text","match":"~2","format":"₂"},{"mode":"text","match":"~3","format":"₃"},{"mode":"text","match":"~4","format":"₄"},{"mode":"text","match":"~5","format":"₅"},{"mode":"text","match":"~6","format":"₆"},{"mode":"text","match":"~7","format":"₇"},{"mode":"text","match":"~8","format":"₈"},{"mode":"text","match":"~9","format":"₉"},{"mode":"text","match":"~+","format":"₊"},{"mode":"text","match":"~-","format":"₋"},{"mode":"text","match":"^0","format":"⁰"},{"mode":"text","match":"^1","format":"¹"},{"mode":"text","match":"^2","format":"²"},{"mode":"text","match":"^3","format":"³"},{"mode":"text","match":"^4","format":"⁴"},{"mode":"text","match":"^5","format":"⁵"},{"mode":"text","match":"^6","format":"⁶"},{"mode":"text","match":"^7","format":"⁷"},{"mode":"text","match":"^8","format":"⁸"},{"mode":"text","match":"^9","format":"⁹"},{"mode":"text","match":"^o","format":"°"},{"mode":"text","match":"^+","format":"⁺"},{"mode":"text","match":"^-","format":"⁻"},{"mode":"text","match":"1/2","format":"½"},{"mode":"text","match":"1/3","format":"⅓"},{"mode":"text","match":"1/4","format":"¼"},{"mode":"text","match":"1/5","format":"⅕"},{"mode":"text","match":"1/6","format":"⅙"},{"mode":"text","match":"1/7","format":"⅐"},{"mode":"text","match":"1/8","format":"⅛"},{"mode":"text","match":"1/9","format":"⅑"},{"mode":"text","match":"1/10","format":"⅒"},{"mode":"text","match":"2/3","format":"⅔"},{"mode":"text","match":"2/5","format":"⅖"},{"mode":"text","match":"3/4","format":"¾"},{"mode":"text","match":"3/5","format":"⅗"},{"mode":"text","match":"3/8","format":"⅜"},{"mode":"text","match":"4/5","format":"⅘"},{"mode":"text","match":"5/6","format":"⅚"},{"mode":"text","match":"5/8","format":"⅝"},{"mode":"text","match":"7/8","format":"⅞"},{"mode":"mark","type":["underline"],"match":["_","_"],"ignoreTrim":true},{"mode":"mark","type":["bold"],"match":["**","**"],"ignoreTrim":true},{"mode":"mark","type":["italic"],"match":["*","*"],"ignoreTrim":true},{"mode":"mark","type":["strikethrough"],"match":["~~","~~"],"ignoreTrim":true},{"mode":"mark","type":["code"],"match":["`","`"],"ignoreTrim":true},{"mode":"block","match":["# "],"type":"heading1"},{"mode":"block","match":["## "],"type":"heading2"},{"mode":"block","match":["### "],"type":"heading3"},{"mode":"block","match":["#### "],"type":"heading4"},{"mode":"block","type":"ul","match":["- "]},{"mode":"block","type":"ol","match":["1. ","1) "]},{"mode":"block","type":"checkbox-item","match":["[] "]}]},"key":"autoformat","handlers":{},"type":"autoformat","inject":{},"editor":{}}],"pluginsByKey":{"react":{"key":"react","type":"react","options":{},"inject":{},"editor":{}},"history":{"key":"history","type":"history","options":{},"inject":{},"editor":{}},"nodeFactory":{"key":"nodeFactory","type":"nodeFactory","options":{},"inject":{},"editor":{}},"event-editor":{"key":"event-editor","handlers":{},"type":"event-editor","options":{},"inject":{},"editor":{}},"inline-void":{"key":"inline-void","type":"inline-void","options":{},"inject":{},"editor":{}},"insertData":{"key":"insertData","type":"insertData","options":{},"inject":{},"editor":{}},"prevSelection":{"key":"prevSelection","handlers":{},"type":"prevSelection","options":{},"inject":{},"editor":{}},"deserializeHtml":{"key":"deserializeHtml","editor":{"insertData":{"format":"text/html"}},"_thenReplaced":1,"type":"deserializeHtml","options":{},"inject":{}},"deserializeAst":{"key":"deserializeAst","editor":{"insertData":{"format":"application/x-slate-fragment"}},"type":"deserializeAst","options":{},"inject":{}},"editorProtocol":{"key":"editorProtocol","type":"editorProtocol","options":{},"inject":{},"editor":{}},"softBreak":{"key":"softBreak","handlers":{},"options":{"rules":[{"hotkey":"shift+enter"}]},"type":"softBreak","inject":{},"editor":{}},"heading1":{"key":"heading1","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h1","H1"]}]},"type":"heading1","options":{},"inject":{},"editor":{}},"heading2":{"key":"heading2","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h2","H2"]}]},"type":"heading2","options":{},"inject":{},"editor":{}},"heading3":{"key":"heading3","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h3","H3"]}]},"type":"heading3","options":{},"inject":{},"editor":{}},"heading4":{"key":"heading4","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["h4","H4"]}]},"type":"heading4","options":{},"inject":{},"editor":{}},"p":{"key":"p","isElement":true,"handlers":{},"options":{"hotkey":["mod+opt+0","mod+shift+0"]},"deserializeHtml":{"rules":[{"validNodeName":"P"}]},"type":"p","inject":{},"editor":{}},"custom1":{"key":"custom1","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom1"}]},"type":"custom1","options":{},"inject":{},"editor":{}},"custom2":{"key":"custom2","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom2"}]},"type":"custom2","options":{},"inject":{},"editor":{}},"custom3":{"key":"custom3","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"custom3"}]},"type":"custom3","options":{},"inject":{},"editor":{}},"imageTitle":{"key":"imageTitle","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"imageTitle"}]},"type":"imageTitle","options":{},"inject":{},"editor":{}},"imageCaption":{"key":"imageCaption","isElement":true,"deserializeHtml":{"rules":[{"validClassName":"imageCaption"}]},"type":"imageCaption","options":{},"inject":{},"editor":{}},"quote":{"key":"quote","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":["blockquote","BLOCKQUOTE"]}]},"type":"quote","options":{},"inject":{},"editor":{}},"combobox":{"key":"combobox","handlers":{},"type":"combobox","options":{},"inject":{},"editor":{}},"mention":{"key":"mention","options":{"trigger":"@","insertSpaceAfterMention":true,"triggerPreviousCharPattern":{},"id":"mention"},"isElement":true,"isInline":true,"isVoid":true,"isMarkableVoid":true,"handlers":{},"plugins":[{"key":"mention_input","isElement":true,"isInline":true,"type":"mention_input","options":{},"inject":{},"editor":{}}],"_thenReplaced":2,"type":"mention","inject":{},"editor":{}},"mention_input":{"key":"mention_input","isElement":true,"isInline":true,"type":"mention_input","options":{},"inject":{},"editor":{}},"bold":{"deserializeHtml":{"rules":[{"validNodeName":["STRONG","B"]},{"validStyle":{"fontWeight":["600","700","bold"]}},{"validNodeName":["SPAN"],"validClassName":"tw-font-bold"}]},"options":{"hotkey":["mod+b","ctrl+b"]},"key":"bold","isLeaf":true,"handlers":{},"type":"bold","inject":{},"editor":{}},"italic":{"deserializeHtml":{"rules":[{"validNodeName":["EM","I"]},{"validStyle":{"fontStyle":"italic"}},{"validNodeName":["SPAN"],"validClassName":"tw-italic"}]},"options":{"hotkey":["mod+i","ctrl+i"]},"key":"italic","isLeaf":true,"handlers":{},"type":"italic","inject":{},"editor":{}},"underline":{"deserializeHtml":{"rules":[{"validNodeName":["U"]},{"validStyle":{"textDecoration":["underline"]}},{"validNodeName":["SPAN"],"validClassName":"tw-underline"}]},"options":{"hotkey":["mod+u","ctrl+u"]},"key":"underline","isLeaf":true,"handlers":{},"type":"underline","inject":{},"editor":{}},"strikethrough":{"key":"strikethrough","isLeaf":true,"handlers":{},"options":{"hotkey":"mod+shift+x"},"deserializeHtml":{"rules":[{"validNodeName":["S","DEL","STRIKE"]},{"validStyle":{"textDecoration":"line-through"}}]},"type":"strikethrough","inject":{},"editor":{}},"a":{"key":"a","deserializeHtml":{"rules":[{"validNodeName":"A"}]},"isElement":true,"isInline":true,"options":{"rangeBeforeOptions":{"matchString":" ","skipInvalid":true,"afterMatch":true},"triggerFloatingLinkHotkeys":"command+k, ctrl+k","keepSelectedTextOnPaste":true},"_thenReplaced":3,"type":"a","inject":{},"editor":{}},"code":{"key":"code","isLeaf":true,"deserializeHtml":{"rules":[{"validNodeName":["CODE"]},{"validStyle":{"fontFamily":"Consolas"}}]},"handlers":{},"options":{"hotkey":"mod+e"},"type":"code","inject":{},"editor":{}},"breakAfterColumn":{"key":"breakAfterColumn","handlers":{},"options":{"columns":1,"gap":"normal","hotkey":["shift+ctrl+enter"]},"type":"breakAfterColumn","inject":{},"editor":{}},"subscript":{"options":{"hotkey":["mod+.","ctrl+."],"clear":"superscript"},"key":"subscript","isLeaf":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":["SUB"]},{"validStyle":{"verticalAlign":"sub"}}]},"type":"subscript","inject":{},"editor":{}},"superscript":{"options":{"hotkey":["mod+,","ctrl+,"],"clear":"subscript"},"key":"superscript","isLeaf":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":["SUP"]},{"validStyle":{"verticalAlign":"super"}}]},"type":"superscript","inject":{},"editor":{}},"align":{"key":"align","inject":{"props":{"validTypes":["heading1","heading2","heading3","heading4","custom1","custom2","custom3","quote","imageCaption","imageTitle","p","p","lic","checkbox-item"],"nodeKey":"align","defaultNodeValue":"left","styleKey":"textAlign","validNodeValues":["left","center","right","justify"]},"pluginsByKey":{"heading1":{"deserializeHtml":{}},"heading2":{"deserializeHtml":{}},"heading3":{"deserializeHtml":{}},"heading4":{"deserializeHtml":{}},"p":{"deserializeHtml":{}},"custom1":{"deserializeHtml":{}},"custom2":{"deserializeHtml":{}},"custom3":{"deserializeHtml":{}},"imageTitle":{"deserializeHtml":{}},"imageCaption":{"deserializeHtml":{}},"quote":{"deserializeHtml":{}}}},"_thenReplaced":3,"type":"align","options":{},"editor":{}},"list":{"options":{"isSoftBreak":false},"key":"list","plugins":[{"key":"ul","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"UL"}]},"type":"ul","options":{},"inject":{},"editor":{}},{"key":"ol","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"OL"}]},"type":"ol","options":{},"inject":{},"editor":{}},{"key":"li","isElement":true,"deserializeHtml":{"rules":[{"validNodeName":"LI"}]},"_thenReplaced":3,"type":"li","options":{},"inject":{},"editor":{}},{"key":"lic","isElement":true,"type":"lic","options":{},"inject":{},"editor":{}}],"type":"list","inject":{},"editor":{}},"ul":{"key":"ul","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"UL"}]},"type":"ul","options":{},"inject":{},"editor":{}},"ol":{"key":"ol","isElement":true,"handlers":{},"deserializeHtml":{"rules":[{"validNodeName":"OL"}]},"type":"ol","options":{},"inject":{},"editor":{}},"li":{"key":"li","inject":{"pluginsByKey":{"deserializeHtml":{"editor":{"insertData":{}}}}},"isElement":true,"deserializeHtml":{"rules":[{"validNodeName":"LI"}]},"_thenReplaced":3,"type":"li","options":{},"editor":{}},"lic":{"key":"lic","isElement":true,"type":"lic","options":{},"inject":{},"editor":{}},"textStyle":{"key":"textStyle","isLeaf":true,"type":"textStyle","options":{},"inject":{},"editor":{}},"checkbox-item":{"key":"checkbox-item","isElement":true,"type":"checkbox-item","options":{},"inject":{},"editor":{}},"indent":{"key":"indent","inject":{"props":{"validTypes":["checkbox-item"],"nodeKey":"indent","styleKey":"marginLeft"}},"handlers":{},"options":{"offset":24,"unit":"px"},"_thenReplaced":3,"type":"indent","editor":{}},"emoji":{"key":"emoji","options":{"id":"emoji","trigger":":","emojiTriggeringController":{"options":{"trigger":":","limitTriggeringChars":2},"_isTriggering":false,"_hasTriggeringMark":false,"text":""}},"_thenReplaced":3,"type":"emoji","inject":{},"editor":{}},"reset-formatting":{"key":"reset-formatting","isElement":false,"type":"reset-formatting","options":{},"inject":{},"editor":{}},"autoformat":{"options":{"rules":[{"mode":"text","match":"--","format":"—"},{"mode":"text","match":"...","format":"…"},{"mode":"text","match":">>","format":"»"},{"mode":"text","match":"<<","format":"«"},{"mode":"text","match":"->","format":"→"},{"mode":"text","match":"<-","format":"←"},{"mode":"text","match":"=>","format":"⇒"},{"mode":"text","match":["<=","≤="],"format":"⇐"},{"mode":"text","match":["(tm)","(TM)"],"format":"™"},{"mode":"text","match":["(r)","(R)"],"format":"®"},{"mode":"text","match":["(c)","(C)"],"format":"©"},{"mode":"text","match":"~0","format":"₀"},{"mode":"text","match":"~1","format":"₁"},{"mode":"text","match":"~2","format":"₂"},{"mode":"text","match":"~3","format":"₃"},{"mode":"text","match":"~4","format":"₄"},{"mode":"text","match":"~5","format":"₅"},{"mode":"text","match":"~6","format":"₆"},{"mode":"text","match":"~7","format":"₇"},{"mode":"text","match":"~8","format":"₈"},{"mode":"text","match":"~9","format":"₉"},{"mode":"text","match":"~+","format":"₊"},{"mode":"text","match":"~-","format":"₋"},{"mode":"text","match":"^0","format":"⁰"},{"mode":"text","match":"^1","format":"¹"},{"mode":"text","match":"^2","format":"²"},{"mode":"text","match":"^3","format":"³"},{"mode":"text","match":"^4","format":"⁴"},{"mode":"text","match":"^5","format":"⁵"},{"mode":"text","match":"^6","format":"⁶"},{"mode":"text","match":"^7","format":"⁷"},{"mode":"text","match":"^8","format":"⁸"},{"mode":"text","match":"^9","format":"⁹"},{"mode":"text","match":"^o","format":"°"},{"mode":"text","match":"^+","format":"⁺"},{"mode":"text","match":"^-","format":"⁻"},{"mode":"text","match":"1/2","format":"½"},{"mode":"text","match":"1/3","format":"⅓"},{"mode":"text","match":"1/4","format":"¼"},{"mode":"text","match":"1/5","format":"⅕"},{"mode":"text","match":"1/6","format":"⅙"},{"mode":"text","match":"1/7","format":"⅐"},{"mode":"text","match":"1/8","format":"⅛"},{"mode":"text","match":"1/9","format":"⅑"},{"mode":"text","match":"1/10","format":"⅒"},{"mode":"text","match":"2/3","format":"⅔"},{"mode":"text","match":"2/5","format":"⅖"},{"mode":"text","match":"3/4","format":"¾"},{"mode":"text","match":"3/5","format":"⅗"},{"mode":"text","match":"3/8","format":"⅜"},{"mode":"text","match":"4/5","format":"⅘"},{"mode":"text","match":"5/6","format":"⅚"},{"mode":"text","match":"5/8","format":"⅝"},{"mode":"text","match":"7/8","format":"⅞"},{"mode":"mark","type":["underline"],"match":["_","_"],"ignoreTrim":true},{"mode":"mark","type":["bold"],"match":["**","**"],"ignoreTrim":true},{"mode":"mark","type":["italic"],"match":["*","*"],"ignoreTrim":true},{"mode":"mark","type":["strikethrough"],"match":["~~","~~"],"ignoreTrim":true},{"mode":"mark","type":["code"],"match":["`","`"],"ignoreTrim":true},{"mode":"block","match":["# "],"type":"heading1"},{"mode":"block","match":["## "],"type":"heading2"},{"mode":"block","match":["### "],"type":"heading3"},{"mode":"block","match":["#### "],"type":"heading4"},{"mode":"block","type":"ul","match":["- "]},{"mode":"block","type":"ol","match":["1. ","1) "]},{"mode":"block","type":"checkbox-item","match":["[] "]}]},"key":"autoformat","handlers":{},"type":"autoformat","inject":{},"editor":{}}},"history":{"undos":[{"operations":[{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"This text is bold.","textStyle":"p","bold":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"This text is italic.","textStyle":"p","italic":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"This text has an underline.","textStyle":"p","underline":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"This text has a strikethrough.","textStyle":"p","strikethrough":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"This text is a code line.","textStyle":"p","code":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"An example of subscript is N"},{"text":"2","subscript":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"An example of superscript is 2"},{"text":"5","superscript":true}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":""}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.","textStyle":"p"}]}},{"type":"remove_node","path":[0],"node":{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number two.","textStyle":"custom1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number one.","textStyle":"custom2"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number three.","textStyle":"custom3"}]},{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two, with more children."}]},{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number two."}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]}]}},{"type":"remove_node","path":[0],"node":{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This comes first."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This comes second."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"And last but not least, this comes third."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 2"}]}]}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":""},{"type":"a","children":[{"text":"This is a Link. Without any breaks before or after"}],"url":"https://frontify.com"},{"text":""}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":"Try mentioning characters, like "},{"type":"mention","category":"user","id":"aaaaaaaa20","children":[{"text":""}]},{"text":" or "},{"type":"mention","category":"group","id":"1111111111","children":[{"text":""}]},{"text":"."}]}},{"type":"remove_node","path":[0],"node":{"type":"p","children":[{"text":""},{"type":"a","chosenLink":{"searchResult":{"link":"https://smartive.ch"},"openInNewTab":true},"children":[{"text":"This is also a Link (Legacy Format)."}]},{"text":""}]}},{"type":"remove_node","path":[0],"node":{"type":"heading1","children":[{"text":"Heading 1","textStyle":"heading1"}]}},{"type":"remove_node","path":[0],"node":{"type":"heading2","children":[{"text":"Heading 2","textStyle":"heading2"}]}},{"type":"remove_node","path":[0],"node":{"type":"heading3","children":[{"text":"Heading 3","textStyle":"heading3"}]}},{"type":"remove_node","path":[0],"node":{"type":"heading4","children":[{"text":"Heading 4","textStyle":"heading4"}]}},{"type":"remove_node","path":[0],"node":{"type":"custom1","children":[{"text":"Custom 1","textStyle":"custom1"}]}},{"type":"remove_node","path":[0],"node":{"type":"custom2","children":[{"text":"Custom 2","textStyle":"custom2"}]}},{"type":"remove_node","path":[0],"node":{"type":"custom3","children":[{"text":"Custom 3","textStyle":"custom3"}]}},{"type":"remove_node","path":[0],"node":{"type":"quote","children":[{"text":"Quote","textStyle":"quote"}]}},{"type":"remove_node","path":[0],"node":{"type":"checkbox-item","checked":true,"indent":0,"children":[{"text":"This is a checked checklist item."}]}},{"type":"remove_node","path":[0],"node":{"type":"checkbox-item","checked":false,"indent":0,"children":[{"text":"This is an unchecked checklist item."}]}},{"type":"remove_node","path":[0],"node":{"type":"checkbox-item","checked":true,"indent":0,"children":[{"text":"This is checked again."}]}},{"type":"remove_node","path":[0],"node":{"type":"checkbox-item","checked":true,"indent":0,"align":"right","children":[{"text":"This is aligned right and checked."}]}},{"type":"insert_node","path":[0],"node":{"type":"p","children":[{"text":"This text is bold.","textStyle":"p","bold":true}]}},{"type":"insert_node","path":[1],"node":{"type":"p","children":[{"text":"This text is italic.","textStyle":"p","italic":true}]}},{"type":"insert_node","path":[2],"node":{"type":"p","children":[{"text":"This text has an underline.","textStyle":"p","underline":true}]}},{"type":"insert_node","path":[3],"node":{"type":"p","children":[{"text":"This text has a strikethrough.","textStyle":"p","strikethrough":true}]}},{"type":"insert_node","path":[4],"node":{"type":"p","children":[{"text":"This text is a code line.","textStyle":"p","code":true}]}},{"type":"insert_node","path":[5],"node":{"type":"p","children":[{"text":"An example of subscript is N"},{"text":"2","subscript":true}]}},{"type":"insert_node","path":[6],"node":{"type":"p","children":[{"text":"An example of superscript is 2"},{"text":"5","superscript":true}]}},{"type":"insert_node","path":[7],"node":{"type":"p","children":[{"text":""}]}},{"type":"insert_node","path":[8],"node":{"type":"p","children":[{"text":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.","textStyle":"p"}]}},{"type":"insert_node","path":[9],"node":{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number two.","textStyle":"custom1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number one.","textStyle":"custom2"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is list item number three.","textStyle":"custom3"}]},{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two, with more children."}]},{"type":"ul","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child of child item number two."}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]}]}},{"type":"insert_node","path":[10],"node":{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This comes first."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This comes second."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"And last but not least, this comes third."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number one."}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Nested child item number two."}]},{"type":"ol","children":[{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 1"}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"Level 5 item 2"}]}]}]}]}]}]}]}]},{"type":"li","children":[{"type":"lic","children":[{"text":"This is aligned right."}],"align":"right"}]}]}},{"type":"insert_node","path":[11],"node":{"type":"p","children":[{"text":""},{"type":"a","children":[{"text":"This is a Link. Without any breaks before or after"}],"url":"https://frontify.com"},{"text":""}]}},{"type":"insert_node","path":[12],"node":{"type":"p","children":[{"text":"Try mentioning characters, like "},{"type":"mention","category":"user","id":"aaaaaaaa20","children":[{"text":""}]},{"text":" or "},{"type":"mention","category":"group","id":"1111111111","children":[{"text":""}]},{"text":"."}]}},{"type":"insert_node","path":[13],"node":{"type":"p","children":[{"text":""},{"type":"a","chosenLink":{"searchResult":{"link":"https://smartive.ch"},"openInNewTab":true},"children":[{"text":"This is also a Link (Legacy Format)."}]},{"text":""}]}},{"type":"insert_node","path":[14],"node":{"type":"heading1","children":[{"text":"Heading 1","textStyle":"heading1"}]}},{"type":"insert_node","path":[15],"node":{"type":"heading2","children":[{"text":"Heading 2","textStyle":"heading2"}]}},{"type":"insert_node","path":[16],"node":{"type":"heading3","children":[{"text":"Heading 3","textStyle":"heading3"}]}},{"type":"insert_node","path":[17],"node":{"type":"heading4","children":[{"text":"Heading 4","textStyle":"heading4"}]}},{"type":"insert_node","path":[18],"node":{"type":"custom1","children":[{"text":"Custom 1","textStyle":"custom1"}]}},{"type":"insert_node"...`

Reply:
Fixed!